recipes-kernel/linux/files/5.x-kernel/0005-feat-add-io-expander-pcal9535-support.patch

```diff
@@ -1,42 +1,82 @@
-From 00a426b081267ea958916fa9cfad707e87c172ef Mon Sep 17 00:00:00 2001
+From 3791a67967bdca96dfcebfc74ead7f547f85d885 Mon Sep 17 00:00:00 2001
 From: "le.jin" <le.jin@siemens.com>
 Date: Wed, 9 Oct 2019 17:08:43 +0800
-Subject: [PATCH 07/14] feat:add io expander pcal9535 support
+Subject: [PATCH 05/14] feat:add io expander pcal9535 support
```

Comment:
Should be replaced with https://github.com/siemens/linux/commit/3ba3ff5c0966280bfd1d40b05b1d7c089f45cc9b - plus other upstream commits if needed.

Reply:
gpio has expose "drive" to  userspace , do the mraa use  this interface or not?

Reply:
There is no such thing as a `drive` property in the kernel's gpio interface (see https://github.com/torvalds/linux/blob/master/Documentation/admin-guide/gpio/sysfs.rst) That extension was a downstream hack of Intel that was never converted into an official feature. We can't carry that unless we start to upstream that generic feature first. After that, drivers can start exposing that feature as well. This patch is a no-go.